production/helm/loki/templates/provisioner/job-provisioner.yaml

```diff
@@ -99,11 +99,12 @@ spec:
               {{- end }}
               {{- $namespace := $.Release.Namespace }}
               {{- with .Values.monitoring.selfMonitoring.tenant }}
+              {{- $secretNamespace := tpl .secretNamespace $ }}
```

Comment:
Not fully sure about this one... Maybe this is also required above line 96. And line 96 needs to reference this var as well.

But only if we expect the user to set a templated var i.e.

```
  provisioner:
    additionalTenants:
      - name: loki
         secretNamespace: {{ .Release.Namespace }}
```

Reply:
yeah, I would be surprised if a user wanted to set this to a templated value. let's wait for a specific request for that functionality before adding it.